docs/csharp/toc.yml

```diff
@@ -222,23 +222,25 @@ items:
     items:
     - name: Overview
       href: linq/standard-query-operators/index.md
-    - name: Filtering Data
+    - name: Store the results of a query in memory
+      href: linq/store-the-results-of-a-query-in-memory.md
```

Comment:
@BillWagner Please verify that this is where the file should be in the TOC (it was orphaned by https://github.com/dotnet/docs/pull/39194).

Reply:
I think a better place would be below the "LINQ to objects" node in line 245.